### PR TITLE
Northwest class 4  Anza-Azam / Northwest class 4 - Anza Azam - HTML/CSS - Week 3

### PR DESCRIPTION
**Pull request by Anza-Azam**
**Volunteers: Are you marking this coursework?** _You can find a guide on how to mark this coursework in `HOW_TO_MARK.md` in the root of this repository_

# Your Details

- Your Name: Anza Azam
- Your City: Northwest
- Your Slack Name: Anza-Azam

# Homework Details

- Module: HTML/CSS
- Week: 3

# Notes

- What did you find easy?
html
- What did you find hard?
Css media queries
- What do you still not understand?
built-in properties for elements
- Any other notes?
Learnt many things while doing this.

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -117,6 +108,8 @@ h2{
     height:auto;
 }
 img:hover{
+	transform: scale(1.2);
+	transition: .3s ease-in-out;
```
Comment (KarenPudner) [inline style.css:112]: I like the hover effects - good use of transition

File: style.css [hunk 2]
```diff
@@ -94,16 +94,7 @@ h2{
...
-    align-items: center;
-}
-.cake-types:nth-child(2){
-    align-items: center;
-}
-.cake-types:nth-child(3){
-    align-items: center;
-}
-.cake-types:last-child{
+.cake-types{
```
Comment (KarenPudner) [inline style.css:97]: Good edit - you have got rid of unnecessary repetition

**Review by KarenPudner — APPROVED** [review]
The webpage looks very good, and meets the resizing criteria. I like the colours, fonts, images and hover effects. It is also good that you are doing small, regular commits. Just one point - when I view the webpage on netify and click the hamburger icon, the menu items are not displayed.

**Comment by Anza-Azam** [comment]
Thank you, Karen, I've added a mobile menu now. Really appreciate your feedback.

**Pull request closed without merging**